packages/common/src/projects/HubProject.ts

```diff
@@ -288,6 +302,28 @@ export class HubProject
       }
     }
 
+    // handle metrics
```

Comment:
not necessarily something to consider as part of this work, but something that's been on my mind is that these `toEditor`/`fromEditor` methods are becoming a bit verbose, and technically to stand this up for initiatives, we'll need to copy this same code into the initiative class...maybe that's not a bad thing, and it allows flexibility for when logic differs between entities, but it would be nice if this entire block was abstracted into a reusable/modular util that's like `handleFromEditorMetrics` so this entire block could be replaced by a single line. Same with the `toEditor` logic above. But I think that's probably unnecessary/over-optimization for now.